.buildkite/scripts/lifecycle/post_command.sh

```diff
@@ -54,6 +54,10 @@ if [[ "$IS_TEST_EXECUTION_STEP" == "true" ]]; then
     ts-node .buildkite/scripts/lifecycle/annotate_test_failures.ts
   fi
 
+  if [[ -d 'target/agent_diagnostics' ]]; then
+    buildkite-agent artifact upload 'target/agent_diagnostics/**/*'
```

Comment:
The entries above this one use `ts-node .buildkite/scripts/lifecycle/annotate_test_failures.ts` after uploading the files... I assume I don't need that here since we're not actually generating `.json` failure files for these, so I did not include it

Reply:
Correct, you don't need that